mosquitto_bridge_registrator.py

```diff
@@ -6,10 +6,9 @@
 import subprocess
 import traceback
 from ve_utils import exit_on_error
-
+VrmNumberOfBrokers = 64
```

Comment:
Shall we make it 128 instead? Is more clicking for me, but it's safer.

Reply:
Done!